Changelog.md

```diff
@@ -1,3 +1,13 @@
+0.10.10.1 <ietf-dane@dukhovni.org> May 2020
+
+ * Fixed off-by-one infinite loop in primMapByteStringBounded.
+ * Documented inadvertent 0.10.6.0 behaviour change in findSubstrings
+ * Fixed findSubString and findSubstrings tests
+ * Work around hPutBuilder test failure for GHC < 7.4
+ * Fixed GHC Travis CI 7.0 builds
+ * Fixed benchmark builds
```

Comment:
This is an unholy nitpicking, but I'd prefer to keep verbs in present tense, consistent with earlier entries.